doc/sphinx/source/conf.py

```diff
@@ -441,6 +442,36 @@
     'sklearn': ('https://scikit-learn.org/stable', None),
 }
 
+# -- Extlinks extension -------------------------------------------------------
+# See https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html
+
+extlinks = {
+    "discussion": (
+        "https://github.com/ESMValGroup/ESMValTool/discussions/%s",
+        "discussion #%s",
```

Comment:
```suggestion
        "Discussion #%s",
```

I think it's nicer to treat this as a proper noun.